scripts/parse_wiki_dumps.py

```diff
@@ -0,0 +1,32 @@
+""" Parsing of Wiki dump and persisting of parsing results to DB. """
+from typing import Optional
+import typer
+from wiki import wiki_dump_api
+
+
+def main(
+    language: str,
+    entity_limit: Optional[int] = typer.Option(None, "--entity_limit"),
+    article_limit: Optional[int] = typer.Option(None, "--article_limit"),
+    alias_limit: Optional[int] = typer.Option(None, "--alias_limit"),
+    use_filtered_dumps: bool = typer.Option(False, "--filter"),
+):
+    """Parses Wikidata and Wikipedia dumps. Persists parsing results to DB.
```

Comment:
It would be good to briefly explain what happens if one of the maximum parse values is reached.

Reply:
Added in next commit. Please close this conversation if you think the comment is sufficient.